src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -135,6 +135,12 @@ private void UpdateRequestFlags(Options options)
                     RequestFlags |= AnalyzeRequestFlags.ShowNonPortableApis;
                 }
 
+                //Set OverwriteOutputFile flag if the output file name is explicitly specified
+                if (options.Out != null)
```

Comment:
I wasn't aware this was the purpose of `AnalyzeRequestFlags`; switched to using a property in `IApiPortOptions`.